src/Chuck.Data/Entities/Weather.cs

```diff
@@ -98,32 +98,30 @@ public Weather(ClientWeatherProto proto)
 
         public bool Update(Weather oldWeather = null)
         {
-            var now = DateTime.UtcNow.ToTotalSeconds();
-            Updated = now;
-            var result = false;
+            Updated = DateTime.UtcNow.ToTotalSeconds();
             if (oldWeather == null)
             {
                 // WebhookController.Instance.AddWeather(this);
-                result = true;
+                return true;
             }
             else if (oldWeather.GameplayCondition != GameplayCondition ||
                 oldWeather.WarnWeather != WarnWeather)
             {
                 // WebhookController.Instance.AddWeather(this);
-                result = true;
+                return true;
             }
-            return result;
+            return false;
         }
 
         public dynamic GetWebhookValues(string type)
         {
             var s2cell = new S2Cell(new S2CellId((ulong)Id));
-            var polygon = new List<List<double>>();
+            var _polygon = new List<List<double>>();
```

Comment:
`_` is for global variables
```suggestion
            var polygon = new List<List<double>>();
```

Reply:
look better

Reply:
Definitely not, just more confusing.
https://stackoverflow.com/questions/9782084

The better fix is to leave it as `polygon` and to remove your "warning" to just specify `polygon` instead of `polygon = polygon,` in the returned dynamic object.

Reply:
this is not global is a var _polygon and set value to stuff null or data (of var)

Reply:
Right, that's why it doesn't need to be prefixed with `_`

Reply:
unused var c => if c never used is _

Reply:
It's a used variable, and unused variables are specifically only `_` without any prefix or suffix. `ThreadPool.QueueUserWorkItem(_ => DoStuff);` <-- This is not the same as what you changed, `polygon` is definitely a referenced variable.

Like I said, change it from `polygon = polygon,` to just `polygon` in the return object and your warning from ReSharper will be happy without this `_` nonsense.

Reply:
polygon = polygon is same uses polygon, //